### PR TITLE
[INLONG-6496][Manager] Support Elasticsearch DataNode

### LINKED ISSUE #6496: [Feature][Manager] Support Elasticsearch datanode
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
- Fixes #6496

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? yes 
  - If yes, how is the feature documented? JavaDocs

### CODE REVIEW

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/node/DataNodeTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+package org.apache.inlong.manager.service.node;
+
+import org.apache.inlong.manager.pojo.node.es.ElasticsearchDataNodeRequest;
+import org.apache.inlong.manager.service.ServiceBaseTest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * Test for data node}
```
Comment (woofyzhao) [inline inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/node/DataNodeTest.java:27]: unnecessary '}'

Comment (vernedeng) [reply inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/node/DataNodeTest.java:27]: fixed, thx

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/node/es/ElasticsearchDataNodeDTO.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    private String httpHosts;
+
+    @ApiModelProperty("audit set name")
+    private String auditSetName;
+
+    /**
+     * Get the dto instance from the request
+     */
+    public static ElasticsearchDataNodeDTO getFromRequest(ElasticsearchDataNodeRequest request) {
+        return ElasticsearchDataNodeDTO.builder()
```
Comment (fuweng11) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/node/es/ElasticsearchDataNodeDTO.java:86]: Recommended use `CommonBeanUtils.copyProperties(request, ElasticsearchDataNodeDTO::new, true);`

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/es/ElasticsearchClusterDTO.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import javax.validation.constraints.NotNull;
+
+/**
+ * Elasticsearch cluster info
+ */
+@Data
+@Builder
+@NoArgsConstructor
+@ApiModel("Elasticsearch cluster info")
+public class ElasticsearchClusterDTO {
```
Comment (healchow) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/es/ElasticsearchClusterDTO.java:37]: This PR is about the Elasticsearch data node, not its cluster.
So I suggest not committing the cluster-related changes to this PR.

Comment (vernedeng) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/es/ElasticsearchClusterDTO.java:37]: fixed, thx

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/node/es/ElasticsearchDataNodeDTO.java (hunk 2 above)
Comment (vernedeng) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/node/es/ElasticsearchDataNodeDTO.java]: fixed, thx

**Review by healchow — APPROVED** [review]
LGTM

**Review by fuweng11 — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by woofyzhao — APPROVED** [review]

**Pull request merged**